src/lib/rest/StringFilter.cpp

```diff
@@ -1587,13 +1662,29 @@ bool StringFilter::qMatch(ContextElementResponse* cerP)
     {
       ContextAttribute* caP = cerP->contextElement.getAttribute(itemP->attributeName);
 
-      if ((itemP->op == SfopExists) && (caP == NULL))
```

Comment:
Yes, I was already starting with subscriptions when we decided to cut the PR in the middle to avoid having the PR too big.
Functests for it have been delayed to the next PR but this change in  the middle of a touched file I  prefer to include (don't want to lose it .. :-))

NTC